<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSerial</class>
 <widget class="QDialog" name="DialogSerial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>582</width>
       <height>410</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabSerial">
      <attribute name="title">
       <string>串口</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="lblPortName">
               <property name="text">
                <string> 串口号</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxPortName">
               <property name="minimumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="lblBaudRte">
               <property name="text">
                <string> 波特率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxBaudRate">
               <property name="minimumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="lblDataBits">
               <property name="text">
                <string> 数据位</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxDataBits">
               <property name="minimumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="lblParity">
               <property name="text">
                <string> 奇偶校验位</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxParity">
               <property name="minimumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="lblStopBits">
               <property name="text">
                <string> 停止位</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxStopBits">
               <property name="minimumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="lblFlowControl">
               <property name="text">
                <string> 流控制</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxFlowControl">
               <property name="minimumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>69</width>
                 <height>20</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>无</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>串口打开:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelOpen">
               <property name="text">
                <string>Unknown</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>串口配置:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelSetting">
               <property name="text">
                <string>Unknown</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>串口发送:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelSend">
               <property name="text">
                <string>Unknown</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>串口接收:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelReceive">
               <property name="text">
                <string>Unknown</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>测试命令</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditTest"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>刷新</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonTest">
               <property name="text">
                <string>测试</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QPushButton" name="pushButtonOpenPort">
               <property name="text">
                <string>打开串口</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonClosePort">
               <property name="text">
                <string>关闭串口</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowser"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QTextBrowser" name="textBrowser_2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
